ts/src/bitmex.ts

```diff
@@ -1873,7 +2022,7 @@ export default class bitmex extends Exchange {
             request['orderID'] = id;
         }
         if (amount !== undefined) {
-            request['orderQty'] = amount;
+            request['orderQty'] = this.amountToPrecision (symbol, amount);
```

Comment:
in the top of the method, you need `this.checkRequiredSymbol('editOrder', symbol);`

Reply:
if user changes amount, then we need to have a symbol, otherwise we can't have correct precision. I don't think it's a pain, as long providing symbol should not be a problem for user.

Reply:
imo, this remains still open. one line above that edit, you might need to `checkReq...(symbol)`. well, if user wants to use ccxt, he has to import ccxt :)
to translate - we can't make it work without it.

Reply:
@ttodua I'm not sure I get your point, `symbol` is already required in `editOrder` so users have to provide it 
![image](https://github.com/ccxt/ccxt/assets/43336371/ae20054d-c671-4b17-953b-0a6c5ce59ba4)

Reply:
ah, you say that signature is enough right? oky.

Reply:
@ttodua yes I mean since it's not an optional argument, it is supposed to be provided